imap_processing/lo/l1a/lo_l1a_write_cdfs.py

```diff
@@ -121,3 +126,158 @@ def create_lo_scide_dataset(sci_de: list):
     )
 
     return sci_de_dataset
+
+
+def create_lo_scicnt_dataset(sci_cnt: list):
+    """
+    Create Lo L1A Science Counts Dataset from the ScienceCounts dataclasses.
+
+    Parameters
+    ----------
+    scide: list
+        List of ScienceCounts data class objects.
+
+    Returns
+    -------
+    xarray.Dataset
+        Lo L1A Science Counts Dataset.
+    """
+    # TODO: How should the times be computed? Using START_A + SHCOARSE for now
+    sci_cnt_times = np.concatenate(
```

Comment:
I don't think you are concatenating here but just creating an array?
```suggestion
    sci_cnt_times = np.array(
```

Below you are doing this already and then adding dtype too, so I think you could add dtype here as well or `.astype(...)`

Reply:
```python
np.concatenate([sci_cnt_data.START_A for sci_cnt_data in sci_cnt])
```

`sci_cnt` is a list of ScienceCount objects and the attributes that I'm pulling from each object are numpy arrays, so without the concatenation this would result in a list of numpy arrays, but I'm trying to concatenate those arrays together. I do need to check with Colin to make sure this is correct for these fields though.

Reply:
Oh interesting, this wasn't clear to me that START_A is an array, that sounds like an individual quantity to me. Maybe it woiuld be helpful to change the variables to START_As plural, or START_A_arr to indicate it is an array/multiple possible values?

Reply:
Yeah, I have information about the shape of those variables in the docstring for the data class: 
```python
"""
START_A : numpy.ndarray
        Single rates for electron, anode A. 2D array Azimuth(6), Energy(7)
"""
```

I'm calling it START_A because that's what is in the packet definition which is also what Lo wants in the L1A CDF as the names for the arrays, so I want to keep the names consistent throughout. Are you ok with that being a little unclear to keep the naming consistent? If there's a question about the data shape, there should be enough information in the class docs.